app/models/concerns/disbursement_costs.rb

```diff
@@ -25,6 +25,11 @@ def total_cost_pre_vat
                             end
   end
 
+  # TODO: update the required adjustments are synced from CW
```

Comment:
**nit** does this need doing or can it be removed or ticket number referenced?

Reply:
That has been removed here https://github.com/ministryofjustice/laa-submit-crime-forms/pull/832